Strategy11/formidable-forms
# Enhanced Placement of Icons in Select Captcha Type Options

This PR aims to improve the placement of icons within the Select Captcha Type options. The proposed changes are part of the Interface Reviews conducted by Razvan.

Figma URL: [Link to Figma Design](https://www.figma.com/file/7Kwlv3H2ShYEI0wekWnPBb/Interface-Review-Fixes?type=design&node-id=702-122&t=DsmaDrn9AMRMkzcv-4)

## QA URL:
https://qa.formidableforms.com/sherv2/wp-admin/admin.php?page=formidable-settings

## Testing Instructions:
1. Go to `WP Admin > Formidable > Global Settings > Captcha` page.
2. Verify that the Select Captcha Type option displays the icon as shown in the "After" screenshot below.

## Screenshots:
### Before
<kbd>
<img width="835" alt="image" src="https://github.com/Strategy11/formidable-forms/assets/69119241/5d4bb55e-37e0-4672-89a2-d6125871d697">
</kbd>

### After
<kbd>
<img width="834" alt="image" src="https://github.com/Strategy11/formidable-forms/assets/69119241/00d9281c-b15b-47ef-aa87-1ce232aa5b52">
</kbd>
